doc/usage.md

```diff
@@ -503,8 +503,11 @@ dns_pending
 name
 :   The user name
 
+pool_size
+:   The user's override pool_size. or NULL if not set.
+
 pool_mode
-:   The user's override pool_mode, or NULL if the default will be used instead.
+:   The user's override pool_mode, or NULL if not set.
```

Comment:
Why change this comment?

Reply:
I thought it would be a good idea to fix also the documentation of `pool_mode`. It was slightly misleading as it is not necessarily true that the default is used when `pool_mode` is not overridden for user. There still can be a setting of `pool_mode` at db level in which case the default is not used.

The same is true for `pool_size` starting this PR so I thought I would make the documentation of the two params practically identical. 

Also the [config.md#L1260-L1261](https://github.com/pgbouncer/pgbouncer/blob/master/doc/config.md?plain=1#L1260-L1261) for the same param is stating this fact correctly already.